workspaces/ui/src/components/dashboards/TestingDashboard.js

```diff
@@ -1,14 +1,12 @@
-import React, { useMemo } from 'react';
+import React, {useMemo} from 'react';
 import Loading from '../navigation/Loading';
-
 // TODO: find a more appropriate place for this logic to live rather than in
 // Contexts now that it's being re-used elsewhere.
-import { stuffFromQueries } from '../../contexts/RfcContext';
-
+import {stuffFromQueries} from '../../contexts/RfcContext';
 // Components and hooks
 // --------------------
-import { Switch, Route, Redirect } from 'react-router-dom';
-import { makeStyles } from '@material-ui/core/styles';
+import {Redirect, Route, Switch} from 'react-router-dom';
+import {makeStyles} from '@material-ui/core/styles';
 import ReportSummary from '../testing/ReportSummary';
```

Comment:
There's a bunch of these reformatting changes in here, I imagine from the built-in WebStorm formatter. We've got a Prettier config file setup as part of the repo, which is the ecosystem defect standard nowadays. To prevent reformatting the codebase all in one go and losing lots of Git history, would it be an idea to [set WebStorm up to use  Prettier](https://prettier.io/docs/en/webstorm.html)? That way we can all format consistently, migrate file by file and end this tug of war 😅.

Reply:
Bingo! Good idea.